### PR TITLE
Added DELETED status for SIM profiles

### PR DESCRIPTION
**Pull request by vihangpatil**

### CODE REVIEW

**Comment by cyriaquebrousse** [comment]
Does this fix the broken event sending as well? I fell out of the discussion this morning, but I like the listener approach :)

**Review by la3lma — APPROVED** [review]
lgtm

File: sim-administration/simmanager/src/main/kotlin/org/ostelco/simcards/admin/SimAdministrationModule.kt [hunk 1]
```diff
@@ -202,5 +202,5 @@ object ResourceRegistry {
 
 object ApiRegistry {
     lateinit var simInventoryApi: SimInventoryApi
-    var simProfileStatusUpdateCallback: ((iccId: String, status: SimProfileStatus) -> Unit)? = null
-}
\ No newline at end of file
+    var simProfileStatusUpdateListeners = mutableListOf<(iccId: String, status: SimProfileStatus) -> Unit>()
```
Comment (cyriaquebrousse) [inline sim-administration/simmanager/src/main/kotlin/org/ostelco/simcards/admin/SimAdministrationModule.kt]: `val`

**Review by cyriaquebrousse — APPROVED** [review]
⛵️

File: sim-administration/simmanager/src/main/kotlin/org/ostelco/simcards/admin/SimAdministrationModule.kt (hunk 1 above)
Comment (vihangpatil) [reply sim-administration/simmanager/src/main/kotlin/org/ostelco/simcards/admin/SimAdministrationModule.kt]: Fixed

**Pull request merged**